### PR TITLE
CEM-2015 rewriteen interface Create and Update..

### PR DESCRIPTION
**Pull request by rmc3408**

### CODE REVIEW

File: src/app/controllers/ModifierController.ts [hunk 1]
```diff
@@ -8,28 +8,11 @@ export interface ModifierChoiceInput {
...
 }
-export interface CreateModifierInput {
-  name: string;
-  identifier: string;
-  description: string;
-  required: boolean;
-  choices: Array<ModifierChoiceInput>;
-  default?: string;
-  default_choices: Array<string>;
+export interface CreateModifierInput extends Partial<Omit<Modifier, keyof DefaultControllerRequired >> {
```
Comment (ralph-dev) [inline src/app/controllers/ModifierController.ts:11]: Partial is only on the deprecated field so you can just ignore the partial or only make that field partial by doing
Partial<Omit<Modifier, keyof defaultController> , 'default'>

**Review by ralph-dev — CHANGES_REQUESTED** [review]
Partial not done correctly for certain types

File: src/app/controllers/ModifierController.ts (hunk 1 above)
Comment (rmc3408) [reply src/app/controllers/ModifierController.ts:11]: Partial only return <T>.

if i make Partial<Omit<Modifier, keyof defaultController> , 'default'>.. give a error.

Comment (ralph-dev) [reply src/app/controllers/ModifierController.ts:11]: Just make default optional on Modifier, then it will propagate :)

**Review by ralph-dev — APPROVED** [review]
LGTM, small commit needed

File: src/app/controllers/ModifierController.ts (hunk 1 above)
Comment (rmc3408) [reply src/app/controllers/ModifierController.ts:11]: Partial<Omit<Modifier, keyof DefaultControllerRequired >, 'default'> 

making default?: string  on Modifier still showing a error.

Comment (rmc3408) [reply src/app/controllers/ModifierController.ts:11]: image

Comment (rmc3408) [reply src/app/controllers/ModifierController.ts:11]: @ralph-dev , take a look in the screeshoot. ?

Comment (ralph-dev) [reply src/app/controllers/ModifierController.ts:11]: There is no error in the screenshot?

**Pull request merged**